src/test/test-seccomp.c

```diff
@@ -805,67 +806,95 @@ static void test_load_syscall_filter_set_raw(void) {
                 assert_se(s = hashmap_new(NULL));
 #if defined __NR_access && __NR_access >= 0
                 assert_se(hashmap_put(s, UINT32_TO_PTR(__NR_access + 1), INT_TO_PTR(-1)) >= 0);
-#else
+                log_debug("has access()");
+#endif
+#if defined __NR_faccessat && __NR_faccessat >= 0
                 assert_se(hashmap_put(s, UINT32_TO_PTR(__NR_faccessat + 1), INT_TO_PTR(-1)) >= 0);
+                log_debug("has faccessat()");
 #endif
 
-                assert_se(seccomp_load_syscall_filter_set_raw(SCMP_ACT_ALLOW, s, SCMP_ACT_ERRNO(EUCLEAN), true) >= 0);
+                if (hashmap_isempty(s)) {
+                        log_info("Neither access() nor faccessat() defined, skipping several tests.");
+                        test_access = false;
+                } else {
```

Comment:
And here too. Every single architecture has `faccessat` defined. But there's also `faccessat2`, which we should filter too. Without that, our test below might fail when glibc switch to use faccessat2.